src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -63,7 +63,9 @@ a#editUncommonSettingsButton:hover{
     border-top: 1px solid #ddd;
     padding-top: 10px;
 }
-
+.border-bottom-dotted {
+    border-bottom: 1px dotted;
+}
```

Comment:
we need dotted underline. And `text-decoration-style` is not supported in most of the browsers.